rdf11.pl

```diff
@@ -1554,6 +1554,10 @@
 rdf_literal(Term) :-
     ground(Term),
     !,
+    (   boolean(Term)
```

Comment:
Thanks.  I don't think the boolean/1 call is right.     A boolean literal is `literal(xsd:boolean, "true|false")`, right?

Reply:
No, but currently `rdf_literal/1` handles both relaxed and canonical forms, so we must handle it properly.

Reply:
Agree.   Sorry.